### DISCUSSION TITLE
change the minimum GPU memory

### DISCUSSION THREAD

**Discussion by hadixcanimm** [seerge/g-helper#1555] [Q&A] [answered]
Is it possible to change the default minimum GPU memory of "-250"? I noticed that lowering this made my device cool down too much.

**Comment by seerge** [comment]
@hadixcanimm default offsets are 0, not -250.

**Reply by hadixcanimm** [reply]
I'm sorry, I didn't explain that right. I was asking for help on how to change the minimum allowed value.

example: gpu memory clock -500

**Comment by hadixcanimm** [comment]
I have already tried a few settings:

"min_gpu_core": -500,
"min_gpu_memory": -500,

But they didn't work.

**Reply by seerge** [reply, marked as answer]
@hadixcanimm it didn't cause those params didn't exist :) i didn't expect such exotic behavior

This build should let you set min values as well
[GHelper.zip](https://github.com/seerge/g-helper/files/13236881/GHelper.zip)

**Reply by hadixcanimm** [reply]
So far, it's working really well. My device is running cooler thanks to you. I don't know why, but the GPU memory clock is not setting to a value lower than -1000. Thanks for providing such a quick solution. Can you make an adjustment that will allow me to set the core clock limit to a value lower than "1000"? I want to run my GPU's core clock at low values like 400MHz to push its limits.

**Reply by seerge** [reply]
@hadixcanimm there is no 1000mhz limitation in the app. Is UI not letting you set lower value or its just not working? if so - then its just nvidia driver’s limit

**Comment by hadixcanimm** [comment]
![sss](https://github.com/seerge/g-helper/assets/114208757/b9dd2bde-6018-487c-9189-b71f0e37ed45)

Is there any way to lower the "1000" limit in the area I highlighted in the screenshot I attached? My device sometimes lowers its clock speed to 300 MHz when it's in a low-power mode. It even caps its clock speed at 1000 MHz when it doesn't need to. It would be really helpful if I could manually set the clock speed.

**Reply by seerge** [reply]
@hadixcanimm this build should allow you to set Clock limit to 400mhz
[GHelper.zip](https://github.com/seerge/g-helper/files/13239612/GHelper.zip)
